discord/ext/pages/pagination.py

```diff
@@ -1035,8 +1098,8 @@ async def edit(
 
         Parameters
         ----------
-        message: :class:`discord.Message`
-            The message to edit with the paginator.
+        target: Union[:class:`~discord.PartialMessage`, :class:`~discord.Message`, :class:`~discord.Interaction`, :class:`~discord.ApplicationContext`, :class:`~discord.ext.bridge.Context`]
+            The target to edit.
```

Comment:
Clarify what we mean here by "target"?

Reply:
Now that I think of it, `message` actually made more sense. All these methods are editing a "message" in a channel. I change this back to `message`